### PR TITLE
Move localization services in Akeneo / PimCatalog components

### PR DESCRIPTION
**Pull request by skeleton**
| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | N
| Added Behats                      | N
| Changelog updated                 | Y
| Review and 2 GTM                  | N
| Micro Demo to the PO (Story only) | N
| Migration script                  | N
| Tech Doc                          | N
```

Move business related to dates & numbers in Akeneo\Component\Localization namespace
Move business related to prices & metrics in Pim\Component\Catalog\Localization namespace

All glue about localization (form, controller, translations, etc) stays in Pim\LocalizationBundle

### CODE REVIEW

File: src/Pim/Bundle/TransformBundle/Denormalizer/Structured/ProductValue/NumberDenormalizer.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Pim\Bundle\TransformBundle\Denormalizer\Structured\ProductValue;
+
+use Akeneo\Component\Localization\Localizer\LocalizerInterface;
+
+/**
+ * Number denormalizer used following attribute types:
```
Comment (pierallard) [inline src/Pim/Bundle/TransformBundle/Denormalizer/Structured/ProductValue/NumberDenormalizer.php]: used *by*

**Comment by pierallard** [comment]
For me it's GTM at now (tiny comment). You just have failing specs, but it can not be a huge refacto.

**Comment by jjanvier** [comment]
bg meuf
GTM with :) and almost <3

**Comment by nidup** [comment]
much :heart: so GTM

**Pull request merged**